src/pages/landing/deriv-go/_deriv-go.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { graphql, useStaticQuery } from 'gatsby'
+import device from 'themes/device'
+import { Container, SectionContainer, Show } from 'components/containers'
+import { Header, Text, QueryImage } from 'components/elements'
+import { localize } from 'components/localization'
+
+const StyledSection = styled(SectionContainer)`
+    background-color: var(--color-white);
+    padding: 8.9rem 0 8rem 0;
+
+    @media ${device.tabletL} {
+        padding: 40px 16px 0;
+    }
+`
+const StyledContainer = styled(Container)`
+    display: flex;
+    flex-direction: column;
+    max-width: 996px;
+
+    @media ${device.tabletL} {
+        width: 100%;
+    }
+`
+const Content = styled.div`
+    width: 100%;
+    max-width: 38.4rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin-right: ${(props) => props.margin_right};
+
+    ${Text} {
+        font-size: 2.4rem;
+        margin-top: 0.8rem;
+
+        @media ${device.tabletL} {
+            text-align: center;
+            font-size: 16px;
+        }
+    }
+
+    @media ${device.tabletL} {
+        margin: 0 auto;
+    }
+`
+
+const ImageWrapper = styled.div`
+    width: 100%;
+    margin-right: ${(props) => props.margin_right};
+
+    @media ${device.tabletL} {
+        margin: 2rem auto;
+    }
+`
+const StyledHeader = styled(Header)`
+    font-size: 32px;
+    line-height: 1.25;
+`
+const StyledText = styled(Text)`
+    font-size: 2.4rem;
+    line-height: 36px;
+    text-align: center;
+    margin-bottom: 4rem;
+`
+const Row = styled.div`
+    flex-direction: ${(props) => props.flex_direction};
+    width: 100%;
+    display: flex;
```

Comment:
```suggestion
const Row = styled(Flex)`
```
You can use the `fd` prop for direction.

Reply:
Amended in the latest commit